install.sh

```diff
@@ -0,0 +1,11 @@
+npm uninstall \
```

Comment:
What's this file for?

Reply:
Not sure, it was added in the first commit before I started working on it. Safe to remove?

Reply:
Yes go for it!